content/reference/v1beta1/application.md

```diff
@@ -58,6 +59,12 @@ Set to `1.14.9`, `1.16.3` or a [blang semver range](https://github.com/blang/sem
 The latest version within the provided range will be used, falling back to the latest version if no version matches.
 Currently `1.14.9` and `1.16.3` are supported, but patch versions may change and newer minor versions may be added in the future.
 
+## kustomizeVersion
+This defaults to `latest`.
+Set to `2.0.3`, `3.5.4` or a [blang semver range](https://github.com/blang/semver#ranges) (like  `>3.0.0 <4.0.0`) to use a specific version of kustomize to apply your app's yaml.
```

Comment:
For now we're just supporting literals (`2.0.3` or `3.5.4`) `latest` and the empty string (which is also `latest`)

Also no fallback yet...